keybinding.go

```diff
@@ -35,6 +35,9 @@ func (kb *keybinding) matchKeypress(key Key, ch rune, mod Modifier) bool {
 // matchView returns if the keybinding matches the current view.
 func (kb *keybinding) matchView(v *View) bool {
 	// if the user is typing in a field, ignore char keys
+	if v == nil {
```

Comment:
might want to combine these two if statements

Reply:
Thanks, I'll do that